scripts/utils-openssl.sh

```diff
@@ -53,7 +54,7 @@ install_openssl() {
 
     if [ ! -d ${OPENSSL_INSTALL_DIR} ]; then
         printf "\tConfigure OpenSSL ${OPENSSL_TAG} ... "
-        ./config shared --prefix=${OPENSSL_INSTALL_DIR} &>> $LOG_FILE
+        ./config shared --prefix=${OPENSSL_INSTALL_DIR} 2>&1 >> $LOG_FILE
```

Comment:
```
In scripts/utils-openssl.sh line 57:
        ./config shared --prefix=${OPENSSL_INSTALL_DIR} 2>&1 >> $LOG_FILE
                                 ^--------------------^ SC2086 (info): Double quote to prevent globbing and word splitting.
                                                        ^--^ SC2069 (warning): To redirect stdout+stderr, 2>&1 must be last (or use '{ cmd > file; } 2>&1' to clarify).
                                                                ^-------^ SC2086 (info): Double quote to prevent globbing and word splitting.
```